src/languages/es.ts

```diff
@@ -6017,6 +6017,11 @@ const translations = {
             part3: '\ny mucho más!',
         },
     },
+    discardChangesConfirmation: {
+        title: '¿Descartar cambios?',
+        body: '¿Estás seguro de que quieres descartar los cambios que hiciste?',
+        confirmText: 'Descartar cambios',
+    },
```

Comment:
Asked for the translation verification here: https://expensify.slack.com/archives/C01GTK53T8Q/p1734322609246139

Reply:
It approved, Let's add the Slack link in author checklist `Link to Slack message:`